apisix/plugins/prometheus/exporter.lua

```diff
@@ -122,6 +122,14 @@ function _M.http_init(prometheus_enabled_in_stream)
             "Etcd modify index for APISIX keys",
             {"key"})
 
+    metrics.shared_dict_capacity_bytes = prometheus:gauge("shared_dict_capacity_bytes",
+            "capacity every moment of nginx shared DICT since APISIX start",
```

Comment:
I am afraid I can't understand what this means...
Could you change the description?